include/seqan3/core/algorithm/concept.hpp

```diff
@@ -6,111 +6,15 @@
 // -----------------------------------------------------------------------------------------------------
 
 /*!\file
- * \brief Provides concepts for the configuration classes.
+ * \brief [DEPRECATED] Provides concepts for the configuration classes.
  * \author Rene Rahn <rene.rahn AT fu-berlin.de>
+ * \deprecated This header will be removed in 3.1. Please use seqan3/core/configuration/detail/concept.hpp instead.
```

Comment:
```suggestion
 * \deprecated This header will be removed in 3.1.0; Please \#include <seqan3/core/configuration/detail/concept.hpp> instead.
```

Reply:
I followed Marcel's example, which he created for us. (math) Now I had to make a lot of changes that did not match his example. Maybe next time you can look over the example as the core team before we take over all the mistakes there.